service/http/test_http_service.py

```
@@ -0,0 +1,51 @@
+        mock_get.return_value = mock_response
+        service = HttpService(base_url="https://api.example.com", endpoint="test")
+
+        #when
+        call_get = lambda : service.get()
+
+        #then
+        with self.assertRaises(Exception):
+            call_get()
+
+    @patch('requests.get')
+    def test_get_with_invalid_url(self, mock_get):
```

Ruff

Missing return type annotation for public function `test_get_with_invalid_url`

Add return type annotation: `None`

(ANN201)